### ISSUE TITLE
Requiem compat: Increasing max health while under the effect of Attrition causes screen twitching

### ISSUE THREAD

**Issue by Derpford** [CleverNucleus/PlayerEx#16]
Essentially, if you increase your max health while under the effect of Attrition (which reduces your max health), the screen twitches constantly. It looks like Attrition and PlayerEX are constantly toggling the player's max health between a higher and lower value, which causes the damage screen tilt to happen rapidly.

**Comment by CleverNucleus** [comment]
Hi there,
Sorry it's taken so long to reply; I've taken a look at Requiem, and (among other things) there is a systematic design flaw with PlayerEx that basically makes these two mods incompatible to a certain level. There's a 1.17.1/1.18 update in the works that should fix this, but there is nothing that can be done in the current version to fix this. Time frame for the next version is early August.
Thanks.

**Comment by Derpford** [comment]
Is there any chance of a backport once the 1.17/1.18 version is out?

**Comment by CleverNucleus** [comment]
Unlikely; the PlayerEx release for versions 1.17/1.18 will be using a more robust (and separate) attributes mod that is being worked on currently. The mod should fix most if not all issues relating to attributes (likely including this one), and allow better compatibility for PlayerEx. Currently, the attributes system PlayerEx employs is a bit haphazard/rubbish - that is to say it is prone to causing issues.

Tl;dr - 1.17+ should have this issue fixed, but only due to an external mod, which means that backporting probably won't happen.
